micromeow/fosterfinder
# Feat/update user

## Description
Added update user flow with global auth handler

## Steps to test
1. Run `npm i`
2. Run `npm run dev`
3. Attempt to login with any provider with a new account (Run `npm run db:reset` to clear your db if you have to)
4. Notice you are directed to the register page
5. Attempt to fill in a name with less than 3 characters to trigger a warning on click of submit
6. Click submit and notice that an error comes up prompting you to fill int he right number of characters
7. Fill in the name and username sections and finish registering
8. You should be directed to the app dashboard with your display name reflected on the page
9. Click on the settings icon on the rightmost corner of the navbar
10. You should be directed to the update profile page
11. Update your username or name
12. You should be able to see the changes reflected in both the database itself (`npm run db:view`) and for display name changes, on the app dashbard
13. Note that the toast to notify user of successful change is in progress in a separate branch `feat/navbar`
